buildstockbatch/gcp/gcp.py

```diff
@@ -610,7 +610,28 @@ def start_batch_job(self, batch_info):
             ),
         )
         instances = batch_v1.AllocationPolicy.InstancePolicyOrTemplate(policy=policy)
-        allocation_policy = batch_v1.AllocationPolicy(instances=[instances])
+        # Make sure the default subnet for this region has access to Google APIs when external IP addresses are blocked.
+        subnet_client = compute_v1.SubnetworksClient()
+        subnet_client.set_private_ip_google_access(
```

Comment:
It seems that some [errors may be in the response](https://cloud.google.com/compute/docs/reference/rest/v1/subnetworks/setPrivateIpGoogleAccess#response-body) and _not_ result in an exception? If so, should probably check and fail (raise an exception)?

Reply:
I added a check, but decided to just log the error, since most times this runs an error here is probably okay. (The setting change should only be needed the first time this runs in a particular region.)